main/src/library/Environment.flix

```diff
@@ -53,6 +53,12 @@ namespace Environment {
             case _:##java.lang.Exception => None
         }
 
+    ///
+    /// Returns the arguments passed to the program via the command line.
+    ///
+    @Experimental
+    pub def getArgs(): Array[String] & Impure = []
```

Comment:
getArgs or getArguments?

Reply:
I also think that we should think about making this a list. 1) you shouldn't change it and 2) you can just call `Array.toList` if you need it for java code or something.

Reply:
Good suggestion. Lets make it into a list. That solves the problem with purity and mutation too!